Library/Homebrew/tap_constants.rb

```diff
@@ -5,4 +5,4 @@
 # match taps' formula paths, e.g. HOMEBREW_LIBRARY/Taps/someuser/sometap/someformula
 HOMEBREW_TAP_PATH_REGEX = Regexp.new(HOMEBREW_TAP_DIR_REGEX.source + %r{/(.*)}.source)
 # match the default and the versions brew-cask tap e.g. Caskroom/cask or Caskroom/versions
-HOMEBREW_CASK_TAP_FORMULA_REGEX = %r{^(Caskroom)/(cask|versions)/([\w+-.]+)$}
+HOMEBREW_CASK_TAP_FORMULA_REGEX = %r{^([Cc]askroom)/(cask|versions)/([\w+-.]+)$}
```

Comment:
Could just make the whole thing case insensitive? Don't mind either way, though.